src/components/HomePage/QuickLinks/index.tsx

```diff
@@ -22,18 +22,18 @@ const QUICK_LINKS = [
     slug: '55',
     key: 'rahman',
   },
-  {
-    slug: '56',
-    key: 'waqiah',
-  },
+  // {
```

Comment:
why are we removing this?

Reply:
yeah we're limiting to only top 5 for now.

Reply:
![image](https://user-images.githubusercontent.com/12745166/143733999-b870bb9a-afaa-47e4-a14a-8dbba3ce12cf.png)

updating this to follow the data from last couple months